doc/cprover-manual/cbmc-unwinding.md

```diff
@@ -4,7 +4,7 @@
 
 ### Iteration-based Unwinding
 
-The basic idea of CBMC is to model the computation of the programs up to
+The basic idea of CBMC is to model the computation of the programs down to
 a particular depth. Technically, this is achieved by a process that
```

Comment:
I would say that the change above is un-idiomatic.

Reply:
I know what you mean. My original edit was simply to change from 'up to' to 'to'. I added the down at the last minute. The basic issue here is that you can't go up to a depth.

Reply:
Would you be ok with just 'to'?

Reply:
Is a bigger rewrite of that sentence on the cards? How about: "The basic idea of CBMC is to model a program's execution up to a bounded number of steps."

Reply:
Thanks, folks. Appreciate your help. I've included that sentence and pushed the file again.